azure-monitor-opentelemetry-distro/tests/configuration/test_configure.py

```diff
@@ -50,14 +54,18 @@ def test_configure_opentelemetry(
         exporter_mock.return_value = exp_init_mock
         resource_init_mock = Mock()
         resource_mock.create.return_value = resource_init_mock
+        sampler_init_mock = Mock()
+        sampler_mock.return_value = sampler_init_mock
         bsp_init_mock = Mock()
         bsp_mock.return_value = bsp_init_mock
-        configure_opentelemetry(
+        configure_azure_monitor(
             connection_string="test_cs",
             disable_tracing=False,
             service_name="test_service_name",
             service_namespace="test_namespace",
             service_instance_id="test_id",
+            sampling_ratio=0.5,
+            tracing_export_interval=15,
```

Comment:
Milliseconds. It's just a value for testing, but figured you may want to change it here, too